Gopkg.toml

```diff
@@ -74,6 +74,10 @@
   name = "github.com/firecracker-microvm/firecracker-go-sdk"
   revision = "961461227bddf7e40a1d690634e866c343910f86"
 
+[[constraint]]
+  name = "github.com/gogo/protobuf"
+  revision = "342cbe0a04158f6dcb03ca0079991a51a4248c02"
```

Comment:
I would expect that after you add the constraint and run `dep ensure -update`, you would have a modified `Gopkg.lock` that also needs to be committed here.

Reply:
> I would expect that after you add the constraint and run `dep ensure -update`, you would have a modified `Gopkg.lock` that also needs to be committed here.

It is already in https://github.com/kata-containers/runtime/blob/ec6a1cc8236c5b7cae1708ad77b22b3aedc338b8/Gopkg.lock#L341
because it's a sub-dependency.

Reply:
Ok I get that, but I was expecting that since you're modifying the `Gopkg.toml`, running `dep ensure -update` would at least modify the digest of the `Gopkg.lock`:
```toml
[[projects]]
  digest = "1:18108594151654e9e696b27b181b953f9a90b16bf14d253dd1b397b025a1487f"
```

Reply:
> Ok I get that, but I was expecting that since you're modifying the `Gopkg.toml`, running `dep ensure -update` would at least modify the digest of the `Gopkg.lock`:
> 
> ```toml
> [[projects]]
>   digest = "1:18108594151654e9e696b27b181b953f9a90b16bf14d253dd1b397b025a1487f"
> ```

Done.

Reply:
Thanks!